Source/Provisioning/ProvisioningGenerator/SmartHotel.IoT.ProvisioningGenerator/Program.cs

```diff
@@ -246,16 +251,23 @@ private void GenerateBrandProvisioningFile( Brand brand, int brandNumber, List<H
 
 				hotelSpaceDescription.AddProperty( new PropertyDescription
 				{ name = PropertyKeyDescription.MaxTemperatureAlertThreshold, value = hotelType.MaxTempAlertThreshold.ToString() } );
-
-				hotelSpaceDescription.AddProperty( new PropertyDescription
-				{ name = PropertyKeyDescription.ImagePath, value = $"{imageAssetsRootPath}hotels/{hotelType.Name.ToLower()}.jpg" } );
-
+				
 				hotelSpaceDescription.AddProperty( new PropertyDescription
 				{ name = PropertyKeyDescription.Latitude, value = hotel.Latitude.ToString() } );
 
 				hotelSpaceDescription.AddProperty( new PropertyDescription
 				{ name = PropertyKeyDescription.Longitude, value = hotel.Longitude.ToString() } );
 
+				hotelSpaceDescription.AddBlob(new BlobDescription
+				{
+					name = $"{brand.Name} {hotel.Name} Blob",
+					type = BlobDescription.FileBlobType,
+					subtype = BlobDescription.NoneBlobType,
+					description = "Hotel image",
+					filepath = $"{ImageFolderRelativePath}/hotels{hotelType.Name.ToLower()}.jpg",

```

Comment:
Missing slash after {ImageFolderRelativePath}/hotels ? Looks like the old code has slash after "hotels" but not before.

Reply:
This isn't including the latest changes to this file. It has been fixed though with a later commit. :)